codecov_auth/views/gitlab.py

```diff
@@ -40,11 +41,16 @@ def get_url_to_redirect_to(self):
         redirect_info = self.redirect_info
         base_url = urljoin(redirect_info["repo_service"].service_url, "oauth/authorize")
         state = self.generate_state()
+
+        scope = get_config("gitlab_scopes", default="api")
```

Comment:
Let's make sure it's set correctly in gitlab config, change this to `get_config("gitlab", "scopes", default="api")` This is how codecov yaml  should look like:
<img width="840" alt="Screenshot 2024-06-12 at 10 49 40 AM" src="https://github.com/codecov/codecov-api/assets/91732700/04fad95e-0820-4bd8-8a5e-ef69fb8bed9f">

Reply:
Trying not to set scopes like this is because this would get parsed as a list when parsing yaml file while when we set the scopes as env var it would be parsed as a string, so don't want to check for both string and list types at run time. So I'm just using a space separated string for setting the scope.